receiver/mongodbatlasreceiver/internal/metric_conversion.go

```diff
@@ -19,15 +19,81 @@ func processMeasurements(
 	var errs error
 
 	for _, meas := range measurements {
-		err := metadata.MeasurementsToMetric(mb, meas, false)
+		err := metadata.MeasurementsToMetric(mb, meas)
 		if err != nil {
 			errs = multierr.Append(errs, err)
 		}
 	}
 
+	err := calculateTotalMetrics(mb, measurements)
+	if err != nil {
+		errs = multierr.Append(errs, err)
+	}
 	if errs != nil {
 		return fmt.Errorf("errors occurred while processing measurements: %w", errs)
 	}
 
 	return nil
 }
+
+func calculateTotalMetrics(
+	mb *metadata.MetricsBuilder,
+	measurements []*mongodbatlas.Measurements,
+) error {
+	var err error
+	dptTotalMeasCombined := false
+	var dptTotalMeas *mongodbatlas.Measurements
+
+	for _, meas := range measurements {
+		switch meas.Name {
+		case "DISK_PARTITION_THROUGHPUT_READ":
+			fallthrough
+		case "DISK_PARTITION_THROUGHPUT_WRITE":
```

Comment:
```suggestion
		case "DISK_PARTITION_THROUGHPUT_WRITE", "DISK_PARTITION_THROUGHPUT_READ":
```

Would it not be simpler to not relying on the fallthrough? I at least think it reads a tad bit better

Reply:
@schmikei can you tell I haven't been writing go consistently in awhile...Looks like a good change to me!